lib/src/drawings.dart

```diff
@@ -52,9 +52,17 @@ class TextDrawing extends GYWDrawing {
   /// If no font is given, it uses the most recent one
   final GYWFont? font;
 
+  /// size of the text (max 48 pt)
+  final int? size;
```

Comment:
I see that we have 3 optional fields now. This API can work but when we actually implement the serialization of the command, we have no choice but to use another control code for this new command. You can't send color without a font prefix because the device will not know which is which. Just a thing to keep in mind.

Also, how is this going to work when TTF font rendering will be deployed? The enum GYWFont doesn't make sense any more because all we will need is the name of the font and a size in pt.

Reply:
@TheToto42 idk if you got a notification for this comment, so I'm tagging you just to be sure

Reply:
The `color` and `size` parameters are there for a future use, so that people are already able to use them, even if this is not working for now. The `GYWFont` will still be used, but will refer to the future TTF font file or something similar. For the color without a font, this is a good remark to keep in mind, we'll see in the future how to handle this, the length of the control message could potentially be used, or we will automatically write the font from the last one that is used.